osh/hub/scan/scanner.py

```diff
@@ -38,6 +39,18 @@ def dig_arch(mock_config):
     return 'noarch'
 
 
+# exist but raises SELinux if it craetes the target directory on its own
```

Comment:
typo here

Reply:
Thank you for spotting it!  Somehow, I have committed an incomplete comment. Fixed in 361bf3c5ba485fa2afa81fdba44966c833caad5e.